package.json

```diff
@@ -7,7 +7,7 @@
     "scripts": {
         "start": "gatsby develop",
         "clean": "gatsby clean",
-        "build": "gatsby build",
+        "build": "gatsby clean && gatsby build",
```

Comment:
A better approach is using `npm run clean` so you ensure running `yarn clean` and `yarn build` use the same steps. Defaulting to `npm run` in package.json means you aren't sneakily requiring yarn.